packages/react-dev-utils/openBrowser.js

```diff
@@ -35,9 +36,27 @@ function getBrowserEnv() {
   } else if (value.toLowerCase() === 'none') {
     action = Actions.NONE;
   } else {
+    // Try detecting browser arguments
+    // Because Chrome for OSX is "google chrome", splitting simply by whitespace is not possible
+    if (value.startsWith(OSX_CHROME)) {
```

Comment:
I believe other software may also contain of two words with spaces. As I suggested in #7226, it would be simplier to add another environment variable for the browser arguments

Reply:
E.g. "google chrome canary"

Reply:
I really wanted this to be a non-invasive solution. I guess not...

Reply:
Isn't there another way, like hardcoding the possible browsers (special cases only)? It's more natural for someone to try passing arguments as one line, instead of using secondary environment variable.

Reply:
You also break the Single Responsibility Principle by mixing browser and it's arguments

Reply:
Another thing is that current implementation might be a breaking change. When you deal with open source, you might want to make sure people are not affected by the things they don't need at all.